### DISCUSSION TITLE
Feeling Lost

### DISCUSSION THREAD

**Discussion by Champ10q** [community/community#147720] [Apps, API and Webhooks]
I have been involved with Cryptocurrency for awhile. I joined through X, telegram, purchased Bitcoin, Adamo, through Kucoin, yet I haven't used a dollar from all my investments. I need to connect all my emails, l
 Internet usage, IP addresses, phone numbers, since 2009. What can I do?

**Comment by ebndev** [comment]
Hi @Champ10q,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this issue. The GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
